packages/modules/pt/src/pageComponents/PropertyFloorDetails.js

```diff
@@ -34,6 +34,23 @@ const PropertyFloorsDetails = ({ t, config, onSelect, formData, userType }) => {
     },
   ];
 
+  const employeeMenu = [
+    {
```

Comment:
For Employee side no of floors is not fixed , it should be fetched from mdms 
only on citizen floor no is fixed

PropertyTax: -> 
Floor

Reply:
done